### PR TITLE
SEO-171911 - ASP.NET MVC Control Missing H1

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Please review the changes for missing H1 below.

Regards,
Sharly A.

### CODE REVIEW

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

File: aspnetmvc/RadialMenu/dimension.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Dimension | Radial Menu |ASP.NET MVC | Syncfusion
-description: dimension
+title: Dimension in ASP.NET MVC Radial Menu Control | Syncfusion
+description: Learn here all about dimension support in Syncfusion ASP.NET MVC Radial Menu Control, its elements, and more.
```
Comment (Girija-Rajendran) [inline aspnetmvc/RadialMenu/dimension.md:4]: remove Syncfusion in the description

Comment (Sharly-Atieno) [reply aspnetmvc/RadialMenu/dimension.md:4]: Done

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**